.travis.yml

```diff
@@ -1,30 +1,30 @@
+sudo: required
```

Comment:
Isn't this ineffective since last year?

Reply:
Yes, since they move to 100% virtualized infra. I will remove it.